src/foam/u2/view/MultiChoiceView.js

```diff
@@ -71,7 +70,27 @@ foam.CLASS({
       `,
       factory: function() {
         return [];
+      },
+      adapt: function(old, nu) {
+        if ( foam.Object.isInstance(nu) && ! Array.isArray(nu) ) {
```

Comment:
foam.Object.isInstance() already checks ! Array. Here's the code:
    function isInstance(o) {
      return typeof o === 'object' && ! Array.isArray(o) &&
          ! foam.core.FObject.isInstance(o) && ! foam.Null.isInstance(o);
    },